### PR TITLE
Add cheatsheet and more videos to README

### LINKED ISSUE #834: Updated README/intro with cheatsheet and more video/slides
There are two videos online that are not on the readme. Also I was wondering if we can make the cheatsheet available from the website more easy to access.

### PR DESCRIPTION
**Pull request by edelarua**
Closes #834

### CODE REVIEW

**Comment by edelarua** [comment]
@Melkiades which 2 videos are you referencing here (https://github.com/insightsengineering/rtables/issues/834#issue-2166301185)?

File: README.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ Packaged releases (both those on CRAN and those between official CRAN
 releases) can be found in the [releases
 list](https://github.com/insightsengineering/rtables/releases).
 
+## Cheatsheet
+
+<a href="https://github.com/insightsengineering/rtables/blob/main/inst/cheatsheet/rtables_cheatsheet_01-24.pdf"><img src="https://raw.githubusercontent.com/insightsengineering/rtables/main/inst/cheatsheet/rtables_cheatsheet_01-24_thumbs.png" width="630" height="247"/></a>
```
Comment (Melkiades) [inline README.md:102]: I think this is much better than the badge I added in {rlistings}. Could you modify that too accordingly?

**Review by Melkiades — COMMENTED** [review]
I thought about it and I think we may want to add it to the top bar or do you think it is too crowded?

Regarding the second half of the issue, there are a couple of videos from Gabe and me that are not present in the readme and I think it could be useful to add them. When you are online I can add them or send them to you ;)

**Review by ayogasekaram — APPROVED** [review]
lgtm!

**Pull request merged**